ember_debug/object-inspector.js

```diff
@@ -842,6 +845,12 @@ function calculateCPs(object, mixinDetails, errorsForObject, expensiveProperties
             tagInfo.tag = metal.track(() => {
               value = calculateCP(object, item.name, errorsForObject);
             });
+            if (!tagInfo.tag.subtag && !tagInfo.tag.subtags && tagInfo.tag === metal.tagForProperty(object, item.name)) {
```

Comment:
I'm not sure what these ifs are for, but it seems like maybe it should be a util function to check for whatever this is checking?

Reply:
if only a tracked is in a tracking context `metal.track(...)`, then the resulting tag will be the same as the `metal.tagForProperty` and will not have any subtag or subtags.
I think emberjs should provide a way to check if a property on an object (or descriptor) is a tracked. Then we can remove this in the feature

Reply:
@chancancode @pzuraq thoughts on this?